mobile/src/store/courses/common.ts

```diff
@@ -1,7 +1,12 @@
 enum ActionType {
+  ADD_COURSE = 'courses/add-course',
+  BECOME_MENTOR = 'course/become-mentor',
+  CREATE_MENTOR = 'course/create-mentor',
```

Comment:
Prefixes for module should be same
```suggestion
  BECOME_MENTOR = 'courses/become-mentor',
  CREATE_MENTOR = 'courses/create-mentor',
```